lib/plugins/extension/lang/el/lang.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/**
+ * @license    GPL 2 (http://www.gnu.org/licenses/gpl.html)
+ *
+ * @author Katerina Katapodi <extragold1234@hotmail.com>
+ */
+$lang['js']['reallydel']       = 'Να γίνει ακύρωση εγκατάστασης αυτής της προέκτασης ';
+$lang['js']['display_viewoptions'] = 'Δέστε Επιλογές ';
+$lang['js']['display_enabled'] = 'ενεργοποιήθηκε';
+$lang['js']['display_disabled'] = 'απενεργοποιήθηκε';
+$lang['js']['display_updatable'] = 'χωρίς δεδομένα';
+$lang['search_for']            = 'Προέκταση Αναζήτησης';
+$lang['search']                = 'Αναζήτηση';
+$lang['extensionby']           = '<strong>%s</strong> με %s	';
+$lang['screenshot']            = 'Εικονίδιο του  %s	';
+$lang['popularity']            = 'Επισκέψεις: %s%%	';
+$lang['homepage_link']         = 'Έγγραφα ';
+$lang['bugs_features']         = 'Λάθη';
+$lang['tags']                  = 'Τίτλοι ;';
+$lang['author_hint']           = 'Αναζητήστε τις προεκτάσεις αυτού του συγγραφέα';
+$lang['installed']             = 'Εγκατάσταση ;';
+$lang['downloadurl']           = 'Κατεβάστε το URL :';
+$lang['repository']            = 'Χώρος αρχείων :';
+$lang['unknown']               = '<em>άγνωστο</em>	';
```

Comment:
I am not sure why strings in this language has weird spaces/tabs afterwards. Is it left by mistake?

Reply:
I fixed it... but I don't speak Greek....

Reply:
Me too :-). I just left that note in case someone or I can fix it later. Thanks for the efforts.

Reply:
@Kastra28 @monotropos

Could you review my changes? comments welcome!